.github/workflows/compare-ds.yaml

```diff
@@ -57,8 +57,15 @@ jobs:
         run: utils/compare_ds.py ssg-${{steps.product.outputs.prop}}-ds.xml build/ssg-${{steps.product.outputs.prop}}-ds.xml | tee diff.log
         env:
           PYTHONPATH: ${{ github.workspace }}
-      - name: Get diff.log
+      - name: Test if there are datastream changes
         if: ${{ steps.ctf.outputs.CTF_OUTPUT_SIZE != '0' }}
+        run: echo "::set-output name=COMPARE_DS_OUTPUT_SIZE::$(stat --printf="%s" diff.log)"
+        id: compare_ds
+      - name: Print changes to datastream if any
```

Comment:
Nitpick: The name gives me the impression that the changes will be added to the datastream.
Ssuggestion: `Print datastream changes if any`

Reply:
Ok, let me quickly rebase and add this modification